installer/bundle/bundle_skel.sh

```diff
@@ -315,7 +315,7 @@ getInstalledVersion()
     if check_if_pkg_is_installed $1; then
         if [ "$INSTALLER" = "DPKG" ]; then
             local version=`dpkg -s $1 2> /dev/null | grep "Version: "`
-            getVersionNumber $version "Version: "
+            getVersionNumber "$version" "Version: "
```

Comment:
I just observed this in a different repo. This alone isn't enough, as the line above will fail as well due to a space in the assignment.